lib/site_prism/error.rb

```diff
@@ -62,4 +62,6 @@ class MissingBlockError < BlockError; end
   # A page was loaded then failed one of the validations defined by the user
   # Formerly known as `NotLoadedError`
   class FailedLoadValidationError < PageLoadError; end
+
+  class UnsupportedElementName < SitePrismError; end
```

Comment:
For learning processes' all errors that are custom should end in `Error`, so this one could be something like `InvalidElementNameError` and would probably subclass `InvalidElementError`

But we can remove it for now, as we won't be throwing an error (yet...)